react/components/SearchBar/components/AutocompleteInput.js

```diff
@@ -23,7 +24,9 @@ export default class AutocompleteInput extends Component {
   }
 
   componentDidMount() {
+    const { runtime } = this.props
     this.changeClassInput()
+    runtime.hints.mobile && this.inputRef.current.focus()
```

Comment:
Is it worth checking if `runtime.hints.mobile` is not undefined to avoid runtime errors?

Reply:
Good idea!